src/server/routes/api/document/reach.js

```diff
@@ -298,8 +307,16 @@ module.exports = {
                       return INTERACTION_TYPE.TRANSCRIPTION_TRANSLATION;
```

Comment:
I wonder whether Reach's 'transcription' events ever have a controller (e.g., TF protein)... If there are only template and product (gene/dna/rna -> rna or protein), then Factoid does not support such reactions at all.

Reply:
We're using an older version of reach, so maybe a newer version handles this case differently.